File: Kotlin/Unit_6/src/example2/Person.java
package example2;

import java.time.LocalDate;

public final class Person {
    private String givenName;
    private String familyName;
    private LocalDate dateOfBirth;
    public String getGivenName() {
        return this.givenName;
    }
    public String getFamilyName() {
        return this.familyName;
    }
    public LocalDate getDateOfBirth() {
        return this.dateOfBirth;
    }
    public Person(String givenName, String familyName, LocalDate dateOfBirth) {
        this.givenName = givenName;
        this.familyName = familyName;
        this.dateOfBirth = dateOfBirth;
    }
}

File: Kotlin/Unit_6/src/example2/Person.kt
package com.makotogo.learn.kotlin.example2

import java.time.LocalDate

class Person(val givenName: String, val familyName: String, val dateOfBirth: LocalDate)

fun main(args: Array<String>) {
    val person = Person(
            "Susan",
            "Neumann",
            LocalDate.of(1980, 3, 17) // 17 Mar 1980
    )

    println("Person: Family Name=${person.familyName}, " +
            "Given Name=${person.givenName}, " +
            "Date of Birth=${person.dateOfBirth}")
}
